week-4-practise/level-2/exercise-1.js

```diff
@@ -11,12 +11,15 @@ We made a start on this for you here:
 var bottle = {
 	volume: 0,
 	fill: function(){
+		this.volume += 100;

```

Comment:
Not quite right, can you spot the bug? What happens if the bottle has half full when you `fill()` it?

Reply:
Oh, thanks for pointing that out, I could do... return (this.volume = this.volume +100); Will make the correction